.github/workflows/ci_cron_weekly.yml

```diff
@@ -70,6 +70,11 @@ jobs:
             python: '3.12'
             toxenv: py312-test-alldeps
 
+          - name: Build Docs with tox
```

Comment:
We used to have this but we removed it from CI because it duplicates RTD PR build. I think once I tried to get rid of `build_docs` altogether in `tox.ini` but some devs apparently use tox to build docs locally. I still not 100% understand why we need to keep it as everyone can run sphinx-build without tox. 🤷‍♀️

Reply:
> I still not 100% understand why we need to keep it as everyone can run sphinx-build without tox.

anything that can be done with tox can be also be done without it, so I guess the answer is "for convenience" ?
Also, this way of doing things is advertised in our dev guide so it'd make sense that it works https://docs.astropy.org/en/latest/development/docguide.html#building-the-documentation-from-source

Reply:
Indeed, I use `tox -e build_cocs` as it is most similar to our old `python setup.py build_docs`...  But agreed that I think we should ensure we either advertise doing the docs build locally like we do on `readthedocs`, or use `tox` on `readthedocs` -- i.e., we should have the same procedure!